controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -495,6 +527,7 @@ func (r *StorageConsumerReconciler) reconcileCephFilesystemSubVolumeGroup(
 	cephFileSystemName string,
 	subVolumeGroupName string,
 	additionalOwner client.Object,
+	forceDelete bool,
```

Comment:
Params should describe desired state, as such the nam of the param should be something like: 
```suggestion
	hasForceDeleteAnnotaion bool,
```